xcode/Subconscious/Shared/Parsers/Subtext.swift

```diff
@@ -658,7 +658,7 @@ extension Subtext {
         // Filter out empty blocks
         let validBlocks = blocks
             .filter { block in !block.isEmpty }
-            .map { block in String(block.body()) }
+            .map { block in String(block.span) }
```

Comment:
We were losing the leading sigils of the excerpt by using `body()`

Reply:
Makes sense to me. I've reverted back to the old style in in the Notebook view and am only rendering the subtext in `StoryEntryView` now, using the new `Subtext.truncate`